specification/sources/chapters/instance.adoc

```diff
@@ -181,7 +181,15 @@ different results if a pname:layerName is available in one call but not in
 another.
 The extensions supported by a layer may also change between two calls, e.g.
 if the layer implementation is replaced by a different version between those
-calls.
+calls, or if an `XrInstance` is created or destroyed between those calls.
+
+This function may: be called before an instance has been created;
+implementations must: not assume an instance exists.
+
+When applications invoke this function, the OpenXR loader's implementation
+is used, which enumerates the registered runtime and API layer manifests;
```

Comment:
Similar to my comment below, the spec does not really know anything about a potential loader's implementation. If we have something that we are going to require here, we need to require it of the runtime or of the API layer.